site/meta.yml

```diff
@@ -29,6 +29,7 @@ toc:
       - event-processing/event-router
       - event-processing/event-streaming-api
       - event-processing/event-filter
+      - event-processing/idempotent-reader
```

Comment:
@krisajenkins -- does `site/meta.yml` need to be updated too (as done here) or just `docs/meta.yml`?

Reply:
Bump @krisajenkins